pkgs/development/web/deno/default.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv, fetchurl, fetchFromGitHub, rust, rustPlatform, python27, installShellFiles }:
+
+let
+  pname   = "deno";
+  version = "1.0.0";
+
+  denoSrc = fetchFromGitHub {
+    owner  = "denoland";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0k8mqy1hf9hkp60jhd0x4z814y36g51083b3r7prc69ih2523hd1";
+
+    fetchSubmodules = true;
+  };
+  cargoSha256 = "1fjl07qqvl1f20qazcqxh32xmdfh80jni7i3jzvz6vgsfw1g5cmk";
+
+  rustyV8Lib = fetchlib "rusty_v8" "0.4.2" {
+    x86_64-linux  = "1ac6kv3kv087df6kdgfd7kbh24187cg9z7xhbz6rw6jjv4ci2zbi";
+    aarch64-linux = "06iyjx4p4vp2i81wdy0vxai2k18pki972ff7k0scjqrgmnav1p8k";
+    x86_64-darwin = "02hwbpsqdzb9mvfndgykvv44f1jig3w3a26l0h26hs5shsrp47jv";
+  };
+
+  arch = rust.toRustTarget stdenv.hostPlatform;
+  fetchlib = name: version: sha256: fetchurl {
+    url    = "https://github.com/denoland/${name}/releases/download/v${version}/librusty_v8_release_${arch}.a";
+    sha256 = sha256."${stdenv.hostPlatform.system}";
+    meta   = { inherit version; };
+  };
+in
+rustPlatform.buildRustPackage rec {
+  inherit pname version cargoSha256;
+
+  src = denoSrc;
+
+  nativeBuildInputs = [
+    python27          # rusty_v8's download script requires python2.7
```

Comment:
Maybe add a link to https://bugs.chromium.org/p/chromium/issues/detail?id=942720 which is the reason why Python 2.7 needs to be used. It will indicate a packager this can be changed once the issue is closed.

Reply:
Cleaned up the explanation & added the tracking link